DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -145,10 +145,11 @@ public void Initial_menu_should_have_list_of_section_card_view_model()
             const string sectionName = "TestSectionName";
             const bool hasLearning = true;
             const double percentComplete = 12.00;
+            const int customisationId = 1;
             var expectedCourseContent = CourseContentHelper.CreateDefaultCourseContent();
             var section = CourseSectionHelper.CreateDefaultCourseSection(sectionName: sectionName, hasLearning: hasLearning, percentComplete: percentComplete);
             expectedCourseContent.Sections.Add(section);
-            var expectedSection = new SectionCardViewModel(section, 1);
+            var expectedSection = new SectionCardViewModel(section, customisationId);

```

Comment:
So the reason we expect it to have this customisation id is because that's the customisation id from the course content. At the moment we're relying on the default value for the customisation id when creating a course content being 1. This makes the test harder to understand and will be harder to debug if it's failing. So instead we should be passing in that customisation id when creating the course content. Does that make sense? 🙂